CRM/Utils/System/Base.php

```diff
@@ -78,15 +78,11 @@ abstract public function loadBootStrap($params = [], $loadUser = TRUE, $throwErr
    *   or equal 0 if not in print mode.
    */
   public static function getContentTemplate($print = 0): string {
-    if ($print === CRM_Core_Smarty::PRINT_JSON) {
-      return 'CRM/common/snippet.tpl';
-    }
-
     switch ($print) {
       case 0:
         // Not a print context.
-        $config = CRM_Core_Config::singleton();
-        return 'CRM/common/' . strtolower($config->userFramework) . '.tpl';
+        // Despite what the template is called
+        return 'CRM/common/CMSPrint.tpl';
```

Comment:
This change would mean that this template file wouldn't be picked up https://github.com/civicrm/civicrm-core/blob/master/templates/CRM/common/joomla.tpl when i think it probably should

Reply:
Ahhhh I always forget Joomla! 
Thanks @seamuslee001 :sweat:

Reply:
I'm looking at the discrepancies between `joomla.tpl` and `CMSPrint.tpl` and wondering if they are mainly because other people have forgotten that separate Joomla template exists when making changes.

I.e. https://github.com/civicrm/civicrm-core/commit/4e6258a7b4171f3db582db272a6231ecb3376eaf - is there any reason this wouldn't make sense on Joomla?

The other substantive changes I can see is there is an extra `crm-content` div wrapper inside the `crm-container` wrapper. Does it do anything? 

And the breadcrumbs seem to come through with {$crumb.url} and {$crumb.title} on Joomla, whereas CMSPrint just outputs { $crumb } .

:thinking:

Reply:
Have added override for Joomla for now though I feel like it would be good if they could be converged.